### PR TITLE
Refactored and Rebranded Code

### PR DESCRIPTION
**Pull request by ribsthakkar**
Renamed Main project under codename Avicena.
Defined model and restructured app in the `avicena` directory.
`experimental` directory now has the old code and serves as a playground/sandbox for trying new things.

### CODE REVIEW

File: alembic/versions/5647b51fe7cb_initial_model_setup.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    sa.Column('assignment_id', sa.Integer(), nullable=False),
+    sa.Column('lats', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('lons', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('trip_ids', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_pu', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_do', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_est_pu', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_sch_pu', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_est_do', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_sch_do', postgresql.ARRAY(sa.Interval()), nullable=True),
```
Comment (rocksolidwebdesign) [inline alembic/versions/5647b51fe7cb_initial_model_setup.py:75]: expand `pu` `do` to `pickup` `dropoff` and or add `sqlalchemy.Column.comment` properties

Comment (ribsthakkar) [reply alembic/versions/5647b51fe7cb_initial_model_setup.py:75]: Addressed in 2b8e7

File: avicena/app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import logging.config
+import yaml
+
+with open('config/log_config.yaml', 'r') as cfg_file:
+    log_config = yaml.load(cfg_file, Loader=yaml.FullLoader)
+    log_config['disable_existing_loggers'] = False
+    logging.config.dictConfig(log_config)
+
+APP_CONFIG_FILE = 'config/app_config.yaml'
+OPTIMIZER_CONFIG_FILE = 'config/optimizer_config.yaml'
```
Comment (rocksolidwebdesign) [inline avicena/app/__init__.py:10]: move all runnable code out of `__init__.py` files, break out into one or more files such as `log_config.py` and or `globals.py` etc

File: avicena/app/__init__.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import logging.config
+import yaml
+
+with open('config/log_config.yaml', 'r') as cfg_file:
+    log_config = yaml.load(cfg_file, Loader=yaml.FullLoader)
+    log_config['disable_existing_loggers'] = False
+    logging.config.dictConfig(log_config)
```
Comment (rocksolidwebdesign) [inline avicena/app/__init__.py:7]: move into a function such as `init_logging` rather than running as a side effect of importing `app`

**Pull request merged**